config/semi_automated.yml

```diff
@@ -449,6 +449,17 @@ parameters:
             tag: 'OAuth CID: 860'
             link: User:Enterprisey/YABBR
             namespaces: [0]
+        effp-helper:
+            regex: '\(\[\[User:Suffusion of Yellow\/effp-helper(\.js)?\|effp-helper\]\]\)'
+            link: User:Suffusion of Yellow/effp-helper
+        Reply link:
+            regex: '\(\[\[w:en:User:Enterprisey\/reply-link\|reply-link\]\]\)'
+            link: User:Enterprisey/reply-link
+            talk_namespaces: true
```

Comment:
I forgot one thing... I think we should add `contribs: true` for Reply link. This will make it still show the edits in the non-automated contributions list. AutoEdits is usually used to filter out the noise of semi-automated edits, and only see relevant substantial contributions. An example is ProveIt; while it is a semi-automated tool, you can still make large content changes which users may wish to see. Similarly users might want to see comments made using Reply link. Hopefully that makes sense... :)

Reply:
@MusikAnimal done